Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -70,6 +70,11 @@
 
   private Settings mSettings;
 
+  private MenuItem mPauseEmulationButton;
+  private MenuItem mUnpauseEmulationButton;
+  private View mPauseEmulationTv;
+  private View mUnpauseEmulationTv;
```

Comment:
To me, it would make more sense if these four didn't exist and we just called `findItem`/`findViewById` in the functions that need to access these. It's a bit hard to follow where and how these get initialized the way it is now.

Reply:
Just to be clear, calling `findItem/findViewById` in both `showPauseEmulationButton()` and `showUnpauseEmulationButton()` while having both of those methods take `Menu` as an argument is more readable? That would duplicate the `findItem/findViewById` for each method and make me add a variable to pass `Menu`, which seems worse to me.

I did find that I can group the variable's assignments together though.

Reply:
Oh, I had missed the part that it was called on `menu` and not just the activity... I suppose it makes sense the way it currently is, then.